src/Psy/ExecutionLoop/Loop.php

```diff
@@ -21,7 +21,7 @@
 class Loop
 {
     /**
-     * The non-forking loop doesn't have much use for Configuration :)
+     * The non-forking loop doesn't have much use for Configuration :).
```

Comment:
What do you mean? the csfixer wants sentence closers, the problem is not the smiley. It currently succesfully builds and is not the smileys fault.